packages/common/src/types.ts

```diff
@@ -43,6 +43,86 @@ export enum StatusEnum {
 export type NUPath = keyof typeof NUPathEnum;
 export type Status = keyof typeof StatusEnum;
 export type Season = keyof typeof SeasonEnum;
+export type SeasonWord = "fall" | "spring" | "summer1" | "summer2";
```

Comment:
did yall do any modifications to the types file? if not then there shouldn't be any changes so maybe consider just copying over the one from main 👀